lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -107,6 +107,10 @@ function dosomething_helpers_preprocess_partners_vars(&$vars) {
       if (!empty($image_node->field_image_landscape)) {
         $img = dosomething_image_get_themed_image($image_node->nid, 'landscape', '550x300');
         $vars['partner_info'][$delta]['image'] = $img;
+      } else {
```

Comment:
I understand that the call to the function is fast. I'm saying I don't think every image call should check for the field and then have to call to get the placeholder image.  the theme shouldn't be responsible for checking that
this should be all that's needed in the theme. 
```
$img = dosomething_image_get_themed_image($image_node->nid, 'landscape', '550x300');
$vars['partner_info'][$delta]['image'] = $img;
```

`dosomething_image_get_themed_image()` should return the image in the crop that's requested or the placeholder image if it doesn't have anything.